apps/editor.planx.uk/src/pages/FlowEditor/components/NewSettings/FlowSettings/Visibility/FlowStatus/components/test/mocks.ts

```diff
@@ -1,131 +1,69 @@
-import { GET_FLOW_STATUS } from "../../queries";
+import type { GetFlowStatus } from "../../types";
 
-export const offlinePublished = [
-  {
-    request: {
-      query: GET_FLOW_STATUS,
-      variables: {
-        flowId: "abc123",
+export const offlinePublished: GetFlowStatus = {
```

Comment:
Nice example of the usefulness of this removal of MockedProvider in favour of msw - we now have a simpler, typed, mock to deal with instead of a more complex GQL mock.